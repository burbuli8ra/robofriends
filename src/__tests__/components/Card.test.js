import React from 'react';
import { shallow } from 'enzyme';
import Card from '../../components/Card';

describe('Card Component', () => {
  it('should render Card component', () => {
    const wrapper = shallow(<Card />);

    expect(wrapper).toMatchSnapshot();
  });
});
