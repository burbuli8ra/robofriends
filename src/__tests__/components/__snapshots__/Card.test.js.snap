// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Component should render Card component 1`] = `ShallowWrapper {}`;
